packages/tokens/properties/color/base.json

```diff
@@ -1,63 +1,4 @@
 {
-  "legacy": {
```

Comment:
I just moved these below the `eds` tokens, which explains some of the changes in `tokens/dist/`